.github/workflows/test_installer.yml

```
@@ -76,16 +76,16 @@ jobs:
     steps:
       - uses: actions/checkout@v3
```

```suggestion
      - uses: actions/checkout@v4
```

Since you have changed in the other jobs as well.